cmd/rm-main.go

```diff
@@ -368,6 +368,11 @@ func listAndRemove(url string, timeRef time.Time, withVersions, isRecursive, isI
 
 		urlString := content.URL.Path
 
+		// rm command is not supposed to remove buckets, ignore if this is a bucket name
```

Comment:
can you give an example where this would happen in the current code?

Reply:
> can you give an example where this would happen in the current code?

mc mb myminio/testbucket1/
mc mb myminio/testbucket2/
mc rm --force --recursive --dangerous myminio/